package.json

```diff
@@ -2,14 +2,20 @@
   "name": "appium-plugins-monorepo",
   "private": true,
   "devDependencies": {
-    "@appium/eslint-config-appium": "4.7.4",
+    "@appium/eslint-config-appium": "5.0.0",
     "@babel/core": "7.16.0",
+    "@babel/eslint-parser": "7.16.3",
     "@babel/plugin-proposal-class-properties": "7.16.0",
-    "appium": "2.0.0-beta.19",
+    "@types/mocha": "9.0.0",
+    "@types/node": "16.11.9",
+    "appium": "2.0.0-beta.22",
```

Comment:
hmm, true, however, if we release an appium version that breaks this repo it will randomly fail an unrelated PR. given that we have renovate bumping things at least 1x/day, this should be OK?